src/data/releases/release.model.ts

```diff
@@ -92,31 +114,38 @@ export type ReleaseFilesInput = {
   removed: string[];
 };
 
+/**
+ * null values are used to clear previously set values
+ */
 export type ReleaseUpdates = {
-  files?: ReleaseFilesInput;
-  label?: string;
-  indices?: string[];
-  publishedAt?: Date;
+  files?: ReleaseFilesInput | null;
+  label?: string | null;
+  indices?: string[] | null;
+  calculatedAt?: Date | null;
+  builtAt?: Date | null;
+  publishedAt?: Date | null;
   state?: ReleaseState;
-  snapshot?: string;
+  error?: string | null;
+  snapshot?: string | null;
 };
```

Comment:
Required to clear data in some release properties when release stages are repeated. Providing `null` as the value is accomplishing this in the `updateRelease()` method.